src/enochecker/checkerservice.py

```diff
@@ -39,15 +39,13 @@
     Required("method", CHECKER_METHODS),  # method to execute
     Required("address", str),  # address to check
     Optional("runId", int, 0),  # internal ID of this run inside our db
-    Optional("team", str, "FakeTeam"),  # the team name
+    Optional("teamName", str, "FakeTeam"),  # the team name
     Optional("teamId", int, 1),  # team ID
-    Optional("round", int, 0),  # which tick we are in
+    Optional("roundId", int, 0),  # which tick we are in
```

Comment:
More to type also... 
And we're in "round 5" not in "round with the id 5" (round id could be a guid or something odd)